File: Python/uberBlack.py
from car import Car


class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, passengers, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver, passengers)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(int id, String license, Account driver, int passengers,
            Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial) {
        super(id, license, driver, passengers);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
